src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goBack(value = 1) {
+      if (value < 1) {
+        return this;
+      }
+      this.coords.y -= value;
+
+      return this;
+    },
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) 

```suggestion
    goBack(value = 1) {
      if (value > 0) {
        this.coords.y -= value;
      }
      
      return this;
    },
```